tests/tasks/airbyte/test_airbyte.py

```diff
@@ -251,3 +251,80 @@ def test_get_job_status_2(self):
         job_id = "1"
         with pytest.raises(JobNotFoundException):
             task._get_job_status(session, airbyte_base_url, job_id)
+
+    # airbyte export tests
+    def test_construction(self):
+        task = AirbyteConfigurationExport()
+        assert task.airbyte_server_host == "localhost"
+        assert task.airbyte_server_port == 8000
+
+    @responses.activate
+    def test_check_health_status(self):
+        airbyte_base_url = "http://localhost:8000/api/v1"
+        responses.add(
+            responses.GET,
+            airbyte_base_url + "/health/",
+            json={"db": True},
+            status=200,
+        )
+        session = requests.Session()
+        task = AirbyteConfigurationExport()
+        response = task._check_health_status(session, airbyte_base_url)
+        assert response
+
+    @responses.activate
+    def test_check_health_status_2(self):
+        airbyte_base_url = "http://localhost:8000/api/v1"
+        responses.add(
+            responses.GET,
+            airbyte_base_url + "/health/",
+            json={"db": False},
+            status=200,
+        )
+        session = requests.Session()
+        task = AirbyteConfigurationExport()
+        with pytest.raises(AirbyteServerNotHealthyException):
+            task._check_health_status(session, airbyte_base_url)
+
+    @responses.activate
+    def test_check_health_status_3(self):
+        airbyte_base_url = "http://localhost:8000/api/v1"
+        responses.add(
+            responses.GET,
+            airbyte_base_url + "/health/",
+            json={"available": True},
+            status=200,
+        )
+        session = requests.Session()
+        task = AirbyteConfigurationExport()
+        response = task._check_health_status(session, airbyte_base_url)
+        assert response
+
+    @responses.activate
+    def test_check_health_status_4(self):
+        airbyte_base_url = "http://localhost:8000/api/v1"
+        responses.add(
+            responses.GET,
+            airbyte_base_url + "/health/",
+            json={"available": False},
+            status=200,
+        )
+        session = requests.Session()
+        task = AirbyteConfigurationExport()
+        with pytest.raises(AirbyteServerNotHealthyException):
+            task._check_health_status(session, airbyte_base_url)
+
+    @responses.activate
+    def test_export_configuration(self):
+        airbyte_base_url = "http://localhost:8000/api/v1"
+        responses.add(
+            responses.POST,
+            airbyte_base_url + "/deployment/export/",
+            body="\x02\x03\x05\x07",
+            status=200,
+        )
+        session = requests.Session()
+        task = AirbyteConfigurationExport()
+        airbyte_config = task._export_configuration(session, airbyte_base_url)
+
+        assert airbyte_config == b"\x02\x03\x05\x07"
```

Comment:
For this test I'd recommend executing `run` instead of `_export_configuration` to ensure that you're testing the public API of the task. That will require an additional responses mock for the health check, so it might be a good idea to turn the health check mock into a pytest fixture.

Reply:
because i established the AirbyteClient, the repetitive health check tests have been eliminated, let me know if the need for fixtures still existed